content/modules/redisgraph/redisgraph-quickstart.md

```diff
@@ -9,10 +9,12 @@ aliases: /rs/getting-started/creating-database/redisgraph-quick-start/
 For this quick start tutorial, you need:
 
 - Either:
+    - A Redis Cloud database [with the RedisGraph module]({{< relref "/rc/administration/setup/create-database.md" >}})
     - A Redis Enterprise Software database [with the RedisGraph module]({{< relref "/modules/add-module-to-database.md" >}})
-    - A Redis Cloud Pro database [with the RedisGraph module](https://redislabs.com/redis-enterprise-cloud/)
 - redis-cli with connectivity to a redis database
 
+You can [set up a free Redis Cloud database]({{< relref "/modules/modules-quickstart.md" >}}) to see the module in action.
```

Comment:
Same. 

Also, the link to "set up a free Redis Cloud database" should link to Redis Cloud, not to another doc page, right?

Reply:
It leads to a doc page specific to setting up Redis Cloud Essentials for a module.